pages/nft/iscn/_iscnId.vue

```diff
@@ -898,20 +902,24 @@ export default class NFTTestMintPage extends Vue {
         await signingClient.setSigner(this.signer)
         logTrackerEvent(this, 'IscnMintNFT', 'CreateNftClass', this.iscnId, 1);
         this.txStatus = TxStatus.PROCESSING
-        const res = await signingClient.createNFTClass(
+
+        const queryClient = signingClient.getISCNQueryClient()
+        const q = await queryClient.getQueryClient()
+        const { classes = [] } = await q.likenft.classesByISCN(this.iscnId).catch(() => ({} as any))
+        const existingClassesLength = classes?.length || 0;
```

Comment:
Any pagination on this API?

Reply:
Yes but for this case we can safely assume <100 so no pages
It is expected there is no existing NFT under the ISCN, if there is we are probably just handling 1-2 wrongly minted classes

Reply:
added comment for this